unic/ucd/tests/category_tests.rs

```diff
@@ -16,10 +16,85 @@ extern crate unic_ucd;
 extern crate unic_utils;
 
 
-use unic_ucd::bidi::BidiClass;
+use unic_ucd::bidi::BidiClass as BC;
 use unic_ucd::normal::is_combining_mark;
+use unic_ucd::category::GeneralCategory as GC;
 use unic_utils::iter_all_chars;
 
+/// `normal::is_combining_mark` and `GeneralCategory::is_mark()` are expected to return
+/// the same results.
+#[test]
+fn test_gen_cat_against_normal() {
```

Comment:
I have added this test to the more-specific component (`ucd-normal`) in <https://github.com/behnam/rust-unic/pull/65>, so you can now drop it from here and have this file only focus on tests for bidi-class against general-category.